shell/platform/glfw/client_wrapper/flutter_window_controller.cc

```diff
@@ -68,6 +68,14 @@ bool FlutterWindowController::CreateWindow(
   return true;
 }
 
+void FlutterWindowController::DestroyWindow() {
```

Comment:
Please update `RunEventLoopWithTimeout` to call this when the window should close, rather than duplicating the cleanup code.

Reply:
Done.